packages/lesswrong/components/BenchmarkComponent.jsx

```diff
@@ -0,0 +1,21 @@
+import { Components, registerComponent } from 'meteor/vulcan:core';
+import gql from 'graphql-tag';
+import { useQuery } from 'react-apollo';
+
+const BenchmarkComponent = () => {
```

Comment:
Is this from a separate branch? Seems sort of random?

Reply:
Whoops, yes, that was a thing hanging around in my working directory that didn't belong in this PR